OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -65,14 +69,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#29r"/>.</note>
                                     <measure type="weight" unit="g">118</measure>
                                 </extent>
-                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay 
+                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library
                                     in January 2020.</foliation>
                                  <collation>
                                     <list>
                                         <item xml:id="q1" n="A">
                                             <dim unit="leaf">2</dim>
                                             <locus from="1r" to="2v"/>
                                         </item>
+                                        <!-- EDS: 06.12.2022 A note on the fact that this is a reused bifolium should be placed here? -->
```

Comment:
yes, please do it, it cannot hurt. We will then later see anyway about the visualisation of the quire structure